extensions/wikia/WikiaMaps/js/WikiaMapsCreateMapTileSet.js

```diff
@@ -103,7 +89,7 @@ define(
 			utils.bindEvents(modal, events);
 
 			// set base step
-			addToStack('selectType');
+			addToStack('browseTileSet');
 
 			// TODO: figure out where is better place to place it and move it there
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS1135)